main.ts

```diff
@@ -342,8 +342,8 @@ const wasm_url = new URL("${wasmFileName}", import.meta.url);
  * @remarks It is safe to call this multiple times and once successfully
  * loaded it will always return a reference to the same object.
  */
-export async function instantiate() {
-  return (await instantiateWithInstance()).exports;
+export async function instantiate(transform) {
```

Comment:
Let’s add a jsdoc param for this so people can understand what the type of this is (same with below).